### PR TITLE
Metric Web-App

### PR DESCRIPTION
**Pull request by ridaarif98**
- Choose an `API` for the web app
### Design
- I follow the design guidelines, including:
  - Colors
  - Layout: composition and space between elements
 ### Interactions
- `Home Page`
  - When the page loads, the web app shows the list of categories that could be filtered out by some parameter
  - When the user clicks (or taps) on a category item, the application navigates to the details page
 - `Detail Page`
   - On the details page, the web app retrieves data from the API to show detailed data in the category.
   - When the user clicks on the "Back" button (<), the user navigates to the home page.
 - Create unit tests for pure functions in your Redux code.
 - Create integration tests for my application using the React Testing Library
 ### Technical requirements
- The project is a single-page application (SPA) built with React and Redux
- The data retrieved from the API is stored in the Redux store
- You can filter the data that you retrieve from the API using a Filter
- Every page has a unique route within the SPA
- The project is deployed and accessible online

### CODE REVIEW

File: .github/workflows/linters.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Linters
```
Comment (arslanbisharat) [inline .github/workflows/linters.yml:1]: - [ ] Your pull request should be from Feature to Development but it is from Development to Main in your case. Please make the changes and request another review.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,82 @@
...
-### Making a Progressive Web App
+Give a ⭐️ if you like this project!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+## Acknowledgments
 
-### Advanced Configuration
+- Microverse for giving me this chance
+- To my Morning Team and coding partners who kept my morale up!
+- The amazing code reviewers for making me improve every day :thumbsup:
```
Comment (arslanbisharat) [inline README.md:81]: - [ ] [Optional] I can see you have worked hard on your Readme but there are Grammar mistakes. please work on those.

File: src/compoenets/renderContinent.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { React, useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { loadAPIData } from '../redux/continent/continent';
+
+const CovidDataRender = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(loadAPIData());
```
Comment (arslanbisharat) [inline src/compoenets/renderContinent.js:8]: - [ ] I can see that the Data is not being rendered properly on the Detail page, Let's make sure APIs are working fine with the app. Let's make sure we use the API that shows details for most of the companies.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/compoenets/renderContinent.js (hunk 3 above)
Comment (ridaarif98) [reply src/compoenets/renderContinent.js:8]: Thank you for the review. I think you click on that country for which there is no cities detail. In this API there are some cities for which there is no detail but for many countries, there are details available.

**Pull request closed without merging**